core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -199,6 +200,9 @@ class KafkaServer(val config: KafkaConfig, time: Time = SystemTime, threadNamePr
         kafkaController = new KafkaController(config, zkUtils, brokerState, kafkaMetricsTime, metrics, threadNamePrefix)
         kafkaController.startup()
 
+        /* start admin manager */
```

Comment:
Technically, we are not starting admin manager here since there is not startup() method.